### PR TITLE
Changes to the example which describes hashing in a log body

### PR DESCRIPTION
**Pull request by rnishtala-sumo**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [ ] Yes, I've run `yarn install`
- [x] No, does not apply to me

## Purpose of this pull request

Changes to the example which describes hashing in a log body

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/send-data/opentelemetry-collector/data-transformations.md [hunk 1]
```diff
@@ -13,8 +13,18 @@ You can find more detailed information about the available OTTL functions and th
 
 ### Example 1: Hashing a password in log body
 
+Consider the following example log
+
+`"username=user password=test"`
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-transformations.md:18]: @rnishtala-sumo should this be inside of the code block?

**Review by kimsauce — CHANGES_REQUESTED** [review]
one question https://github.com/SumoLogic/sumologic-documentation/pull/3332/files#r1403683142

File: docs/send-data/opentelemetry-collector/data-transformations.md (hunk 1 above)
Comment (rnishtala-sumo) [reply docs/send-data/opentelemetry-collector/data-transformations.md]: thanks! put this inside a code block

Comment (kimsauce) [reply docs/send-data/opentelemetry-collector/data-transformations.md]: @rnishtala-sumo Oh sorry, to clarify, I was asking if that line belonged in the code block (above the `processors` line), or if it's meant to stand alone like that.

**Review by kimsauce — CHANGES_REQUESTED** [review]
[one Q](https://github.com/SumoLogic/sumologic-documentation/pull/3332#discussion_r1408767542)

File: docs/send-data/opentelemetry-collector/data-transformations.md (hunk 1 above)
Comment (rnishtala-sumo) [reply docs/send-data/opentelemetry-collector/data-transformations.md]: its meant to be a separate line

File: docs/send-data/opentelemetry-collector/data-transformations.md [hunk 2]
```diff
@@ -27,16 +37,22 @@ processors:
         statements:
           - set(attributes["password"], SHA256(attributes["password"]))
           - set(attributes["password"], Concat(["passwd", attributes["password"]], "="))
-          - replace_pattern(attributes["message"], "password=([0-9A-Za-z]+_)", attributes["password"])
+          - replace_pattern(body, "password=([0-9A-Za-z]+_)", attributes["password"])
```
Comment (sumoanema) [inline docs/send-data/opentelemetry-collector/data-transformations.md:40]: Should we delete the attribute "message" after we are done with it ?

Comment (rnishtala-sumo) [reply docs/send-data/opentelemetry-collector/data-transformations.md:40]: good point, pushed a change to use `delete_key`

File: docs/send-data/opentelemetry-collector/data-transformations.md [hunk 3]
```diff
@@ -13,8 +13,18 @@ You can find more detailed information about the available OTTL functions and th
 
 ### Example 1: Hashing a password in log body
 
+Consider the following example log
+
+```username=user password=test```
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-transformations.md:18]:
```suggestion
`username=user password=test`
```

File: docs/send-data/opentelemetry-collector/data-transformations.md (hunk 1 above)
Comment (kimsauce) [reply docs/send-data/opentelemetry-collector/data-transformations.md]: thanks for confirming!

File: docs/send-data/opentelemetry-collector/data-transformations.md [hunk 4]
```diff
@@ -13,8 +13,18 @@ You can find more detailed information about the available OTTL functions and th
 
 ### Example 1: Hashing a password in log body
 
+Consider the following example log
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-transformations.md:16]:
```suggestion
Consider the following example log.
```

**Review by kimsauce — APPROVED** [review]

**Pull request merged**